src/main/java/edu/iu/uits/lms/lti/controller/RedirectableLtiController.java

```diff
@@ -61,7 +61,7 @@ protected String performMacroVariableReplacement(String inputUrl) {
       String sisUserId = oidcTokenUtils.getSisUserId();
       String familyName = oidcTokenUtils.getPersonFamilyName();
       String givenName = oidcTokenUtils.getPersonGivenName();
-      String[] roles = oidcTokenUtils.getRoles();
```

Comment:
Go back to the service and make the getRoles (and others) method names more meaningful